docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,73 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after a configuration update
+
+The update of an existing Elasticsearch cluster configuration can fail because the operator is unable to apply the changes required while replacing the pods of a given Elasticsearch cluster.
+
+This can be identified by the `AGE` of the pod being older than the configuration update, or the `STATUS` of the pods:
+
+[source,sh]
+----
+kubectl get pods
+
+NAME                      READY   STATUS    RESTARTS   AGE
+quickstart-es-default-0   1/1     Running   0          15m
+quickstart-es-default-1   1/1     Running   0          15m
+quickstart-es-default-2   1/1     Running   0          15m
+----
+
+Possible causes include:
+
+* The Elasticsearch cluster is not healthy.
++
+[source,sh]
+----
+kubectl get elasticsearch
+
+NAME         HEALTH   NODES   VERSION   PHASE   AGE
+TODO: Finish example
+----
++
+In this case, you have to check and fix your shard allocations.
+For more information, see link:https://www.elastic.co/guide/en/elasticsearch/reference/7.10/cluster-health.html[Cluster health API]
+and link:https://www.elastic.co/guide/en/cloud-on-k8s/1.3/k8s-deploy-elasticsearch.html#k8s_monitor_cluster_health_and_creation_progress[Monitor cluster health and creation progress].
```

Comment:
```suggestion
In this case, you have to link:{ref}/cluster-allocation-explain.html[check] and fix your shard allocations.  The link:{ref}/cluster-health.html[cluster health], link:{ref}/cat-shards.html[cat shards], and
and link:https://www.elastic.co/guide/en/cloud-on-k8s/1.3/k8s-deploy-elasticsearch.html#k8s_monitor_cluster_health_and_creation_progress[Monitor cluster health and creation progress] APIs can assist in tracking the shard recover process.
```

Reply:
- [ ] Fix reference to ECK documentation

Reply:
The page id is `{p}-deploy-elasticsearch` for <https://www.elastic.co/guide/en/cloud-on-k8s/1.3/k8s-deploy-elasticsearch.html>, but not sure how to handle the `#k8s_monitor_cluster_health_and_creation_progress` portion of the URL.